### PR TITLE
test for round moves

### PR DESCRIPTION
**Pull request by zeitlinger**
still WIP - trying to see if CI also gets `TypeError: Cannot read property 'center' of undefined`

apparently the same error on CI

@coyotte508 do you have an idea how to fix this?

We could also move recent.ts to engine if nothing else works.

### CODE REVIEW

File: viewer/src/logic/recent.spec.ts [hunk 1]
```diff
@@ -1,64 +1,47 @@
...
-      want: string[];
+      give: { moveHistory: string[] };
+      want: { recentMoves: string[]; roundMoves: string[] };
     }[] = [
       {
         name: "player starts",
         give: {
           moveHistory: [
             "init 2 randomSeed2",
-            "p2 rotate",
```
Comment (coyotte508) [inline viewer/src/logic/recent.spec.ts:17]: this is removed, but below it's in `want.roundMoves`

**Comment by coyotte508** [comment]
The tests try to build a mine in sector 8 in a two players game, where there are only 7 sectors.

I think I will add back the possibility to do tests on the viewer like on the engine, for pure logic tests (without components), that will also allow the possibility to launch a test file in the debugger. Unfortunately I only put the VS Code configuration in the repo in `.vscode/launch.json`, no idea how to do that in IntelliJ.

File: viewer/src/logic/recent.spec.ts (hunk 1 above)
Comment (zeitlinger) [reply viewer/src/logic/recent.spec.ts:17]: the PR is not ready - just wanted to get the viewer tests to run..

**Comment by coyotte508** [comment]
Can this be closed?

**Comment by zeitlinger** [comment]
> Can this be closed?

you said:

> I think I will add back the possibility to do tests on the viewer like on the engine

I think that would be a good idea - but it's not urgent.

**Comment by coyotte508** [comment]
The command was added in https://github.com/boardgamers/gaia-project/commit/739dbcb8eee8a76ba873c3d77569807e3cc8fc73

**Comment by zeitlinger** [comment]
> The command was added in [739dbcb](https://github.com/boardgamers/gaia-project/commit/739dbcb8eee8a76ba873c3d77569807e3cc8fc73)

ok, closing the PR then

**Pull request closed without merging**